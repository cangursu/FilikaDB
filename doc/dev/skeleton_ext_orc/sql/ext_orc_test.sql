CREATE EXTENSION ext_orc;
SELECT ext_orc_scan('/home/postgres/projects/filikadb/doc/dev/skeleton_ext_orc/TestOrcFile.test1.orc');
SELECT ext_orc_content('/home/postgres/projects/filikadb/doc/dev/skeleton_ext_orc/TestOrcFile.test1.orc');