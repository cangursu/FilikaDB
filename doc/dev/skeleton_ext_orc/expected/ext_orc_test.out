CREATE EXTENSION ext_orc;
SELECT ext_orc_scan('/home/postgres/projects/filikadb/doc/dev/skeleton_ext_orc/TestOrcFile.test1.orc');
                                             ext_orc_scan                                             
------------------------------------------------------------------------------------------------------
 ext_orc_sca       :                                                                                 +
 orcFile           :  /home/postgres/projects/filikadb/doc/dev/skeleton_ext_orc/TestOrcFile.test1.orc+
 orcFile Len       :  79                                                                             +
 ORCScanFile       :                                                                                 +
 filename          :  /home/postgres/projects/filikadb/doc/dev/skeleton_ext_orc/TestOrcFile.test1.orc+
 cwd               :  /usr/local/pgsql/data                                                          +
 Rows              :  2                                                                              +
 Batches           :  1                                                                              +
 
(1 row)

SELECT ext_orc_content('/home/postgres/projects/filikadb/doc/dev/skeleton_ext_orc/TestOrcFile.test1.orc');
                                                                                                                                                                                                                                                         ext_orc_content                                                                                                                                                                                                                                                         
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 ext_orc_content   :                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            +
 orcFile           :  /home/postgres/projects/filikadb/doc/dev/skeleton_ext_orc/TestOrcFile.test1.orc                                                                                                                                                                                                                                                                                                                                                                                                                           +
 orcFile Len       :  79                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        +
 ORCPrintContents  :                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            +
 filename          :  /home/postgres/projects/filikadb/doc/dev/skeleton_ext_orc/TestOrcFile.test1.orc                                                                                                                                                                                                                                                                                                                                                                                                                           +
 Line0:            :  {"boolean1": false, "byte1": 1, "short1": 1024, "int1": 65536, "long1": 9223372036854775807, "float1": 1, "double1": -15, "bytes1": [0, 1, 2, 3, 4], "string1": "hi", "middle": {"list": [{"int1": 1, "string1": "bye"}, {"int1": 2, "string1": "sigh"}]}, "list": [{"int1": 3, "string1": "good"}, {"int1": 4, "string1": "bad"}], "map": []}                                                                                                                                                            +
 Line1:            :  {"boolean1": true, "byte1": 100, "short1": 2048, "int1": 65536, "long1": 9223372036854775807, "float1": 2, "double1": -5, "bytes1": [], "string1": "bye", "middle": {"list": [{"int1": 1, "string1": "bye"}, {"int1": 2, "string1": "sigh"}]}, "list": [{"int1": 100000000, "string1": "cat"}, {"int1": -100000, "string1": "in"}, {"int1": 1234, "string1": "hat"}], "map": [{"key": "chani", "value": {"int1": 5, "string1": "chani"}}, {"key": "mauddib", "value": {"int1": 1, "string1": "mauddib"}}]}+
 
(1 row)

